loops_lab.js

```diff
@@ -0,0 +1,197 @@
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Good demonstration of for and while loops overall. I noted few minor errors but looks good overall. Keep up the good work!